### ISSUE TITLE
Compatibility with Tortoise 0.17.0

### ISSUE THREAD

**Issue by marcoaaguiar** [tortoise/aerich#136]
Hi,

Is the current version compatible with the new release of tortoise-orm?

I can't install because of version lock on the last aerich release.
Is it a matter of just bumping the version?

If not, is there an expected date or any pending issue that need to be solved?

Currently there is a conflict in depencies resolution:

```
  Because no versions of aerich match >0.5.1,<0.6.0
   and aerich (0.5.1) depends on tortoise-orm (>=0.16.21,<0.17.0), aerich (>=0.5.1,<0.6.0) requires tortoise-
orm (>=0.16.21,<0.17.0).
  And because tortoise-orm (0.17.0) depends on tortoise-orm (0.17.0)
   and no versions of tortoise-orm match >0.17.0,<0.18.0, aerich (>=0.5.1,<0.6.0) is incompatible with tortoi
se-orm (>=0.17.0,<0.18.0).
  So, because test depends on both tortoise-orm (^0.17.0) and aerich (^0.5.1), version solving failed.
 ```

**Comment by long2ice** [comment]
just update version

**Comment by marcoaaguiar** [comment]
Aerich 0.5.2, which changes the dependency to `tortoise-orm = "*"`, is not out yet.
I will install it directly from GitHub in the meantime

**Comment by marcoaaguiar** [comment]
Closed by Release of 0.5.2
